launch-dedicated.sh

```diff
@@ -18,5 +18,5 @@ while true; do
      mono --debug OpenRA.Server.exe Game.Mod=$Mod \
      Server.Name="$Name" Server.ListenPort=$ListenPort Server.ExternalPort=$ExternalPort \
      Server.AdvertiseOnline=$AdvertiseOnline \
-     Server.EnableSingleplayer=$EnableSingleplayer Server.Password=$Password
+     Server.EnableSingleplayer=$EnableSingleplayer Server.Password=$Password | tee /var/log/openra/server.log
```

Comment:
This won't work if the `/var/log/openra/` directory hasn't first been created.

Reply:
That's true, sorry about that. Perhaps add a folder named logs to base dir and have that as the default one?